actionpack/CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+*   Raise error on unpermitted open redirects.
+
+    Add `fallback_location` and `allow_other_host` options to `redirect_to`.
+    Opt into this behaviour with `ActionController::Base.raise_on_open_redirects = false`.
```

Comment:
Unless I'm reading the code wrong, this should either say

```suggestion
    Opt in to this behaviour with `ActionController::Base.raise_on_open_redirects = true`.
```

or

```suggestion
    Opt out of this behaviour with `ActionController::Base.raise_on_open_redirects = false`.
```

Reply:
🤦 I updated the property name but not the value. Thanks!